zcncore/transaction_query_mobile.go

```diff
@@ -489,3 +489,73 @@ func GetInfoFromAnySharder(urlSuffix string, op int, cb GetInfoCallback) {
 
 	cb.OnInfoAvailable(op, StatusSuccess, string(qr.Content), "")
 }
+
+func GetUserLockedTotal(clientID string) (int64, error) {
```

Comment:
I saw this function in `transaction_query.go` as well, if the function signatures are the same, then we should move it to `transaction_query_base.go` to avoid duplicate code.

Reply:
There was no transaction_query_base.go file, so created it & moved the method to it from transaction_query.go.